gui/ColorizeSurfaceOptions.cxx

```diff
@@ -307,6 +307,7 @@ void ColorizeSurfaceOptions::colorize()
       {
         d->UI.comboBoxColorDisplay->setEnabled(true);
         emit colorModeChanged(d->UI.buttonGroup->checkedButton()->text());
+        this->InsideColorize = false;
```

Comment:
Somewhat OT as this existed previously, but why are there a bunch of protected members that don't follow the naming convention when this class has a PIMPL?